packages/client/test/integration/Subscriber.test.ts

```diff
@@ -47,7 +48,7 @@ describe('Subscriber', () => {
     
             const sub = await subscriber.subscribe(stream.id)
     
-            const publisherNode = environment.startNode(publisherWallet.address)
+            const publisherNode = environment.startNode(publisherWallet.address as NodeID)
```

Comment:
With NodeID no longer being (necessarily) an Ethereum address, would it be better for the tests to reflect this fact? Right now the tests may give the impression that Wallet has something to do with NodeID while in reality they are separate concepts.

Reply:
I'll make a separate PR about this